core/src/main/java/com/datastrato/gravitino/listener/impl/EventListenerManager.java

```diff
@@ -0,0 +1,155 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.listener.impl;
+
+import com.datastrato.gravitino.listener.EventListenerPlugin;
+import com.datastrato.gravitino.listener.SupportsAsync;
+import com.datastrato.gravitino.listener.SupportsAsync.Mode;
+import com.datastrato.gravitino.utils.MapUtils;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Joiner;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * EventListenerManager loads listeners according to the configurations, and assemble the listeners
+ * with following rules:
+ *
+ * <p>Wrap all listener with EventListenerWrapper to do some common process, like exception handing,
+ * record metrics.
+ *
+ * <p>For async listeners with the shared dispatcher, will create a default AsyncQueueListener to
+ * assemble the corresponding EventListenerWrappers.
+ *
+ * <p>For async listeners with the isolated dispatcher, will create a separate AsyncQueueListener
+ * for each EventListenerWrapper.
+ */
+public class EventListenerManager {
+  private static final Logger LOG = LoggerFactory.getLogger(EventListenerManager.class);
+  public static final String GRAVITINO_EVENT_LISTENER_PREFIX = "gravitino.eventListener.";
+  static final String GRAVITINO_EVENT_LISTENER_NAMES = "names";
+  @VisibleForTesting static final String GRAVITINO_EVENT_LISTENER_CLASSNAME = "className";
```

Comment:
"class" should be enough, no need to add "Name".

Reply:
done